### PR TITLE
Update react-native to 0.59.10

### PR DESCRIPTION
**Pull request by liulianci**
looks like react native 0.55.3 is not working with my xcode(10.2), update react-native to 0.59.10

### CODE REVIEW

**Comment by friederbluemle** [comment]
Looks like Travis is still failing:

```
yarn install v1.19.1
[1/4] Resolving packages...
[2/4] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://repository.cache.walmart.com/content/groups/npm/prop-types/-/prop-types-15.7.2.tgz: getaddrinfo ENOTFOUND repository.cache.walmart.com repository.cache.walmart.com:443".
```

Make sure to undo everything in `yarn.lock` that changed from registry.yarnpkg.com to repository.cache.walmart.com

**Comment by friederbluemle** [comment]
Looking good already. I assume the updates to `project.pbxproj` are related to and required for using Xcode 10.2? If that's the case, can you also update `.travis.ci` in the same commit?

```
-      osx_image: xcode9.3
+      osx_image: xcode10.2
```

File: ios/ElectrodeReactNativeBridge.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1126,4 +1139,4 @@
 /* End XCConfigurationList section */
 	};
 	rootObject = DA4BE0B31E36DBCD00DBA1ED /* Project object */;
-}
+}
```
Comment (friederbluemle) [inline ios/ElectrodeReactNativeBridge.xcodeproj/project.pbxproj]: If this was automatically removed, can you please manually add the now-missing newline at EOF?

**Comment by friederbluemle** [comment]
Actually, can you just bump it up to `osx_image: xcode10.3.1`? That'll line it up with the simulator image that's already being used. At the same time, can you also change `iPhone 6` to `iPhone 8`?

**Comment by liulianci** [comment]
@friederbluemle yes I can change to xcode 10.3. I checked this repo on Friday and I thought I saw we are supporting xcode 9.0+. but I cannot find it now. maybe I was blind on Friday

**Comment by friederbluemle** [comment]
Yikes.. I guess I should have checked the Travis CI log more carefully. It failed but exited with 0, making the CI status green (success) 😢

**Pull request merged**